libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -52,13 +52,20 @@ export const NavbarMainItems = styled('div')`
   font-size: 1.125rem;
   text-transform: uppercase;
 
-  ${({theme}) => css`
+  ${({theme, show}) => css`
     ${theme.breakpoints.up('sm')} {
-      display: grid;
+      display: ${show ? 'none' : 'grid'};
     }
   `}
 `
 
+const IconButtonWrapper = styled('div')`
```

Comment:
```suggestion
export const NavbarIconButtonWrapper = styled('div')`
```

We want to export so it can be re-used/overwritten